docs/reference/contributing/target/bootloader.md

```diff
@@ -0,0 +1,17 @@
+## Bootloader
+
+Arm Mbed OS allows you to update your firmware. Our Pelion Device Management platform enables this capability. The following documentation includes instructions and examples on how to update firmware on a new target.
```

Comment:
Nitpick, but I don't think Pelion Device Management is the platform. Pelion Device Management is one of the services available under the Pelion IoT Platform umbrella.